integrationtests/src/test/java/io/kroxylicious/proxy/Utils.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.proxy;
+
+import io.kroxylicious.proxy.config.ConfigParser;
+import io.kroxylicious.proxy.config.Configuration;
+
+public class Utils {
+    public static KafkaProxy startProxy(String config) throws InterruptedException {
```

Comment:
Not in scope for this PR, but we have KroxyConfigBuilder that could replace this.